datastream-kafka/src/main/java/com/linkedin/datastream/kafka/KafkaProducerWrapper.java

```diff
@@ -75,7 +75,8 @@
   private static final AtomicInteger NUM_PRODUCERS = new AtomicInteger();
   private static final Supplier<Integer> PRODUCER_GAUGE = NUM_PRODUCERS::get;
 
-  private static final int CLOSE_TIMEOUT_MS = 2000;
+  private static final int DEFAULT_PRODUCER_CLOSE_TIMEOUT_MS = 10000;
```

Comment:
Is 10 seconds sufficient? Should this be longer to give more time to drain records?

Reply:
I will be setting 60 sec via config for MM cases. The default is for change capture case. Along with this, in places where flushlessEventProducer is enabled, I will be setting the producerFlush timeout as well to 60 sec.